Avoid-Roadblocks.md

```diff
@@ -1,5 +1,5 @@
-* The issue you file must fulfill the requirements of [Bug report Template](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) to provide greater insight and understanding of the issue. By providing detailed information the issue will be investigated and resolved in a timely manner.
```

Comment:
Maybe we should have a separate PR for updating all the WCT `master` branch references to `main`?

Reply:
I like the idea but we do not have many changes in the wiki in regards to Master VS Main so it should do the job for now. In the future will have to be careful though once the Wiki is more scalable.